apps/deploy-web/src/context/WalletProvider/WalletProvider.tsx

```diff
@@ -314,6 +315,7 @@ export const WalletProvider = ({ children }) => {
         isCustodial: !isManaged,
         isWalletLoading: isLoading,
         isTrialing: isManaged && !!managedWallet?.isTrialing,
+        isOnboarding: isManaged && !managedWallet?.stripeCustomerId,
```

Comment:
question (blocking): shouldn't this be to check if the user has a user account?

Reply:
There are 3 user wallet states:
- anonymous trial account with just a managed wallet
- user account with email with a managed wallet, still with trial
- user account with email with a managed wallet, with added funds with credit card

Reply:
Having a wallet doesn't mean that a user has a user account. We allow users to start trial anonymously and it creates a userSetting row in db without `userId` so we need to check if the user has `.userId` defined.

Reply:
For reference: https://github.com/akash-network/console/blob/ad173e2f74248f2acb73d81a61065f0360c0cd80/apps/deploy-web/src/hooks/useUser.ts#L18